.github/workflows/CI.yml

```
@@ -38,8 +38,14 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
-      - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
+      - uses: julia-actions/julia-buildpkg@latest
+      - name: Run tests 
+        shell: julia --color=yes --project=. {0}
+        run: |
+          using Pkg
+          # force it to use this PR's version of the test utilities subpackage
+          Pkg.develop(PackageSpec(path=joinpath(".", "lib", "AbstractFFTsTestUtils")))
+          Pkg.test(coverage=true)
```

There's no reason to not use the test action. We can just add the subpackage in a separate step:
```suggestion
      - name: Install dependencies
        run: julia --project=. -e 'using Pkg; Pkg.develop(PackageSpec(path=joinpath(pwd(), "lib", "AbstractFFTsTestUtils"))); Pkg.instantiate()'
      - uses: julia-actions/julia-runtest@v1
```